valhalla/sif/dynamiccost.h

```diff
@@ -538,6 +542,8 @@ class DynamicCost {
             // We are in range at the time we are allowed at this edge
             if (access_type == baldr::AccessType::kTimedAllowed)
               return true;
+            else if (access_type == baldr::AccessType::kDestinationAllowed)
+              return allow_conditional_destination_restrictions_ || is_dest;
```

Comment:
I'd rename it to `allow_conditional_destination_` - it directly says: we allow go through edges with `conditional=destination` tag.
Current name confuse me a little, looks like we allow conditional destination restrictions to use=)

Reply:
It's better, done